pkg/controller/upgradeconfig/config.go

```diff
@@ -10,17 +10,24 @@ type config struct {
 }
 
 type upgradeWindow struct {
-	TimeOut int `yaml:"timeOut" default:"60"`
+	TimeOut int `yaml:"timeOut" default:"120"`
+	DelayTrigger int `yaml:"delayTrigger" default:"30"`
 }
 
 func (cfg *config) IsValid() error {
 	if cfg.UpgradeWindow.TimeOut <= 0 {
 		return fmt.Errorf("Config upgrade window time out is invalid")
 	}
-
+	if cfg.UpgradeWindow.DelayTrigger <= 0 {
```

Comment:
Should this be allowed to be set to 0? This could be a way to disable the functionality.

Reply:
Thanks for the suggestion - I've updated it so that if the delay trigger is 0, the checking for a delay is ignored (/never notifies of a delay).